nanovg-sys/build.rs

```diff
@@ -7,7 +7,12 @@ use std::process::Command;
 fn build_library(backend_macro: &str) {
     let target = env::var("TARGET").unwrap();
     let mut config = cc::Build::new();
-    config.warnings(false); // Hide the nanovg warnings. Not really relevant to us.
+
+    // Hide the nanovg warnings. Not really relevant to us.
+    // cc::Build::warnings(false); // this does not disable warnings, it can be used only for enabling them
```

Comment:
Is that true? I remember `config.warnings(false)` was working when I added it. Could it be possible that this behaviour changed when we updated gcc to cc?

Reply:
Yes, I looked through cc-rs code and even checked whether gcc-rs with version 0.3 had different behavior, but warnings function did not change, it never disabled any warnings, only added. You can find old source [here](https://docs.rs/crate/gcc/0.3.54/source/src/lib.rs) and new one [here](https://github.com/alexcrichton/cc-rs/blob/master/src/lib.rs#L1233), it only adds new flags to the compiler if warnings are set, if they are not set, it does not do anything.